apps/pie-docs/src/engineers/getting-started/structure.md

```diff
@@ -6,6 +6,44 @@ eleventyNavigation:
     order: 2
 ---
 
-## Project Structure
+## PIE Component Systems
 
-Coming soon…
+Currently, JET has several officially supported component systems that implement PIE.
+
+- [Fozzie](https://vue.pie.design/) – a set of Vue components created by the legacy Just Eat web team
+- [Snacks](https://snacks.takeaway.com/) – a set of React components created by the legacy Takeaway design system team
+- [Skip PIE Project](https://react.pie.design/) – a set of React components created by the Skip web team
```

Comment:
I feel like it doesn't matter too much who made them, rather what they are used for?
```suggestion
- [Fozzie](https://vue.pie.design/) – a set of Vue components for legacy Just Eat web applications
- [Snacks](https://snacks.takeaway.com/) – a set of React components for legacy Takeaway.com web applications
- [Skip PIE Project](https://react.pie.design/) – a set of React components for legacy SkipTheDishes web applications
```

Reply:
I think this one should stay as it is. Teams outside of Just Eat use the Vue components, and Snacks may be used outside of Takeaway.com platforms shortly (Scoober are talking about using some of them for instance).